pyproject.toml

```diff
@@ -64,6 +64,12 @@ docs = [
 Source = "https://github.com/matrix-nio/matrix-nio"
 Documentation = "https://matrix-nio.readthedocs.io/en/latest/"
 
+[tool.setuptools.packages.find]
+where = ["src"]
```

Comment:
Was this needed?

Reply:
no, but I was scratching my head why it even worked. I think explicit is better than implicit.